kratos/sources/model_part_io.cpp

```diff
@@ -797,7 +797,9 @@ namespace Kratos
                 bool temp;
 
                 ReadWord(value); // reading value
-                ExtractValue(value,temp);
+                std::stringstream str_stream(value);
```

Comment:
what are u doing here? why is this needed?

in any case please add a test for the new feature, ideally it should fail before your PR

Reply:
ok, we'll put the test. @farrufat-cimne , any help is welcome.

Reply:
Do you want to read booleans for variable-data?

If I am not completely wrong then I think I used this before
I think with `1` and `0` though

Reply:
@philbucher , according to @farrufat-cimne , we had problems at reading 0 and 1 yesterday, but we will make sure by writing the test proposed by @RiccardoRossi

Reply:
most importantly, the ExtractValue function should be doing exactly what you added there.

did u try to use 

      ExtractValue<bool>(....)

Reply:
Not exactly, there's an extra order added. Let's check the test first, because maybe this dicsussion is useless. Need more time.

Reply:
@philbucher  was right and it works perfectly with 1 and 0.
I just pushed the test, and undid the previous changes.